terratorch/cli_tools.py

```diff
@@ -98,9 +98,32 @@ def save_prediction(prediction, input_file_name, out_dir, dtype:str="int16"):
     file_name = os.path.basename(input_file_name)
     file_name_no_ext = os.path.splitext(file_name)[0]
     out_file_name = file_name_no_ext + "_pred.tif"
-    logging.info(f"Saving output to {out_file_name} ...")
+    logger.info(f"Saving output to {out_file_name} ...")
     write_tiff(result, os.path.join(out_dir, out_file_name), metadata)
 
+def import_custom_modules(custom_modules_path:None | Path | str =None) -> None:
+
+    if custom_modules_path:
+
+        custom_modules_path = Path(custom_modules_path)
+
+        if custom_modules_path.is_dir():
+
+            # Add 'custom_modules' folder to sys.path
+            workdir = custom_modules_path.parents[0]
+            module_dir = custom_modules_path.name
+
+            sys.path.append(workdir)
+
+            try:
+                importlib.import_module(module_dir)
+                logger.info(f"Found {custom_modules_path}")
+            except Exception:
```

Comment:
Could you find out what specific exception you would get in this case and catch that? Because at the moment, you'll catch all exceptions and then give it the new message below, which might not be correct. Also, once caught, use a `from` statement.

Reply:
What about these choices. 

```
    if custom_modules_path:

        custom_modules_path = Path(custom_modules_path)

        if custom_modules_path.is_dir():

            # Add 'custom_modules' folder to sys.path
            workdir = custom_modules_path.parents[0]
            module_dir = custom_modules_path.name

            sys.path.append(workdir)

            try:
                importlib.import_module(module_dir)
                logger.info(f"Found {custom_modules_path}")
            except ImportError:
                raise ImportError(f"It was not possible to import modules from {custom_modules_path}.")
        else:
            raise ValueError(f"Modules path {custom_modules_path} isn't a directory. Check if you have defined it properly.")
    else:
        logger.info("No custom module is being used.")

```